client/components/profilePage/myProfile.jsx

```diff
@@ -23,26 +33,25 @@ export default function myFriends({user = {friends: []}}) {
       </div>
       <div id='friendsLinks'>
         <a>Friends</a>
-        <a variant='primary' onClick={() => { setShow(true) }}>Find Friends</a>
-        <AllUsersModal handleClose={() => { setShow(false) }} show={show} />
-        <a>View All Friends</a>
+        <a variant='primary' onClick={() => { setAllUsersShow(true) }}>Find Friends</a>
+        <AllUsersModal handleClose={() => { setAllUsersShow(false) }} show={allUsersShow} user={user} />
+        <a variant='primary' onClick={() => { setAllFriendsShow(true) }}>View All Friends</a>
+        <AllFriendsModal handleClose={() => { setAllFriendsShow(false) }} show={allFriendsShow} friendsList={friendsList}/>
       </div>
       <div id='myFriends'>
-        {
-        user.friends.map((friend) => {
-
-          return (
-            <div className = 'friendsDiv' key = {friend.username}>
-              <div><img src={image1}></img></div>
-              <div>{friend.username}</div>
-            </div>
-          )
-        })
+        {friendsList.length > 0 &&
+          friendsList.map((friend) => {
+            return(
+              <div className = 'friendsDiv' key = {friend.username}>
+                <div className = 'friendsBody'><img className='friendsImage' src={friend.photoUrl}></img></div>
```

Comment:
equals does not need to have spaces around it when adding something like a className